src/chrome/buildContextMenus.js

```diff
@@ -28,19 +28,17 @@ const buildContextMenus = (store, chrome) => {
     const root = chrome.contextMenus.create({ // Menu
       title: option.title,
       contexts: ['browser_action'],
-    },
-    () => {
```

Comment:
do we not need to wait until `create` is done before creating the submenus?

Reply:
Hm, so this change I added in the last commit because it was causing the only failures after upgrading to Babel 7. The issue I was seeing was
```
ReferenceError: Cannot access 'root' before initialization
```
@prayogoa is there a better way of resolving that?

Reply:
Looks like it's not needed after all, the only dependency with the submenu is with the parent's menu id, which is guaranteed to be available as soon as `.create` returns